.github/workflows/check-code.yaml

```diff
@@ -127,3 +127,32 @@ jobs:
             echo "::error::due to bandit violations ${ret##*:}"
             exit 1
           fi
+  check_code_pytest:
```

Comment:
@AJThorpe I think we should create a separate workflow for this, maybe unit-tests.yaml?

Reply:
If we do move this to a different file, maybe this file should be renamed to `lint.yaml`. I think we might end up with:
- `run_linters.yaml`
- `run_unit_tests.yaml`
- `run_integration_tests.yaml`

Having separation will let us more easily control when we want different tests to run.

What do you think of the file naming @mindyls?

Reply:
I think
run-linters.yaml
run-unit-tests.yaml
run-integration-tests.yaml
I just renamed to make them all have '-' in the name!

Reply:
@AJThorpe do you want to rename check-code.yaml to run-linters.yaml, then create a new run-unit-tests.yaml file, and move this pytest block to that file?  We do not have any integration tests yet.

Reply:
@AJThorpe @jsolbrig I think we should remove this block of code from the workflow, so only the xr_to_dtree.py, test_dtree.py, and release note files are updated.  We'll address the workflows in a separate PR.

Once this file is reverted, we should be able to just merge this PR with the basics of datatrees implemented.

Reply:
Sounds good to me.